physical/raft/raft.go

```diff
@@ -118,14 +118,28 @@ type LeaderJoinInfo struct {
 	// LeaderCACert is the CA cert of the leader node
 	LeaderCACert string `json:"leader_ca_cert"`
 
-	// LeaderClientCert is the client certificate for the follower node to establish
-	// client authentication during TLS
+	// LeaderClientCert is the client certificate for the follower node to
+	// establish client authentication during TLS
 	LeaderClientCert string `json:"leader_client_cert"`
 
-	// LeaderClientKey is the client key for the follower node to establish client
-	// authentication during TLS
+	// LeaderClientKey is the client key for the follower node to establish
+	// client authentication during TLS.
 	LeaderClientKey string `json:"leader_client_key"`
 
+	// LeaderCACert is the CA cert of the leader node. This should only be
+	// provided via Vault's configuration file.
+	LeaderCACertFile string `json:"leader_ca_cert_file"`
+
+	// LeaderClientCert is the client certificate for the follower node to
+	// establish client authentication during TLS. This should only be provided
+	// via Vault's configuration file.
+	LeaderClientCertFile string `json:"leader_client_cert_file"`
+
+	// LeaderClientKey is the client key for the follower node to establish
+	// client authentication during TLS. This should only be provided via
+	// Vault's configuration file.
+	LeaderClientKeyFile string `json:"leader_client_key_file"`
```

Comment:
The comments on these new fields have the wrong field name (missing File at the end) and should mention that this is a path to a file on disk.